src/kaltura-player.js

```diff
@@ -78,6 +79,8 @@ class KalturaPlayer extends FakeEventTarget {
    */
   _autoPaused: boolean = false;
 
+  RemotePlayerType: Object = RemotePlayerType;
```

Comment:
not needed. use import like:
```
import {cast} from 'kaltura-player-js;
const {RemotePlayerType} =  cast;
```

Reply:
how do I access it from ui then ?

Reply:
I need to expose it so ui can access it though

Reply:
like the code I wrote

Reply:
already exposed 
https://github.com/kaltura/kaltura-player-js/blob/master/src/index.js#L51
https://github.com/kaltura/kaltura-player-js/blob/master/src/common/cast/index.js#L25